genai-function-calling/spring-ai/src/main/resources/application.yml

```diff
@@ -4,6 +4,13 @@ spring:
     web-application-type: none
     banner-mode: "off"
   ai:
+    # Right now, we cannot select azure or openai with a single property.
+    # See https://github.com/spring-projects/spring-ai/issues/2712
+    model:
+      embedding:  ${spring.ai.model.chat}
```

Comment:
so, for now, I'm re-using the chat value for the others that have auto-configuration depenedencies despite being used. At some point I think the word "model" shouldn't be used as these are inference providers, not models